website/docs/components/carousel/carousel-a11y.md

```diff
@@ -13,7 +13,7 @@ Table: Keyboard support
 | Key                         | Function                                                                                                                                                                           |
 | --------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `Tab`, `Shift + Tab`        | Moves focus through interactive elements in the carousel.                                                                                                                          |
-| `Enter`, `Space`            | Opens slide in modal window if `zoom` property is set to `true`.                                                                                                                   |
+| `Enter`                     | Opens slide in modal window if `zoom` property is set to `true`.                                                                                                                   |
 | `Left Arrow`, `Right Arrow` | Moves focus to the previous or next tab in the slide indicators. If focus is on the last tab, moves focus to the first tab. Shows the slide associated with the newly focused tab. |
```

Comment:
```suggestion
| `Left Arrow`, `Right Arrow` | Activates the previous or next tab in the slide indicators. If the last tab is active, activates the first tab. Shows the slide associated with the activated tab. |
```
I think it's better to avoid the word "focus" here, because as I see the focus is always on the entire indicator row. And maybe ever remove the last sentence, it might be redundant.

Reply:
Yeah! You are definitely right here, thank you ✨